praw/models/reddit/emoji.py

```diff
@@ -162,8 +225,21 @@ def add(self, name, image_path):
             )
         response.raise_for_status()
 
+        if not isinstance(post_flair, bool):
+            post_flair = True
+        if not isinstance(user_flair, bool):
+            user_flair = True
+        if not isinstance(mod_only, bool):
+            mod_only = False
```

Comment:
These checks are unnecessary. Please remove these lines and just use the values given in the `data` dict on the lines below.

Reply:
10-4

Reply:
I didn't realize there was a page that showed in-line comments. Cool. *The more you know.*